#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_man.files.dpatch by  <sgran@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add FILES to manpage

@DPATCH@

--- hdparm/hdparm.8.old	2005-11-21 23:40:00.000000000 +0000
+++ hdparm/hdparm.8	2005-11-21 23:40:18.000000000 +0000
@@ -551,6 +551,8 @@
         m       maximum security
 
 .B THIS FEATURE IS EXPERIMENTAL AND NOT WELL TESTED. USE AT YOUR OWN RISK.
+.SH FILES
+/etc/hdparm.conf
 .SH BUGS
 As noted above, the
 .B -m sectcount
